cli/hack/test-e2e.sh

```diff
@@ -1,11 +1,18 @@
 #!/bin/bash
 
-set -e -x -u
+set -e -x -o pipefail
```

Comment:
Just to be sure, we are removing `-u` because we do not want to error out if ns env variable is not set correct?

Reply:
Yep, that's correct.